tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1454,6 +1454,31 @@ Here are some examples of expressions::
 Identifiers, attributes, and subscripts are valid assignment targets. Other expressions cannot be on the left side of
 an assignment.
 
+self
+^^^^
+
+``self`` can be used to refer to the current instance and is often equivalent to directly referring to symbols available in
+the current script, however, it also allows you to access properties, methods, and other names in subclasses of the current script.
```

Comment:
You probably meant "subtypes". "Subclasses" can be confused with inner classes. It is better to use the term "inherited class", "child class" or "descendant class".

Reply:
You're right, sorry! It's probably my Python bias again :p